models/comments.models.js

```diff
@@ -16,3 +16,15 @@ exports.selectCommentsByArticleId = (article_id) => {
       return rows;
     });
 };
+
+exports.prepComment = (comment) => {
```

Comment:
insert, add or create may be a little more descriptive than prep here for what this function is doing